### DISCUSSION TITLE
nuxt3按照npm文档使用该插件没有自动生成.pem文件

### DISCUSSION THREAD

**Discussion by lianggegege** [liuweiGL/vite-plugin-mkcert#97] [Q&A]
我用的nuxt3项目和最新的vite-plugin-mkcert包，本地执行启动命令只生成了certs目录，没有生成对应的cert.pem和dev.pem文件，导致报错，并且使用了该插件后多了一条警告The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.

**Comment by liuweiGL** [comment]
1. 个包根据同时配置了 cjs 跟 esm 的入口，关于警告你应该参考官方文档检查项目是否为 esm 语法
2. 把 debug 打开看下日志，根据目前的信息无法判断为什么不生成证书文件
